src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/QueryString.java

```diff
@@ -58,7 +58,8 @@ static boolean checkJobWithQuery(File job, String queryString) {
             }
 
             for (int i = 0; i < cachedQueryList.size(); i++) {
-                if (jobArray.length < i) {
+                System.out.println(jobArray.length + " " + i);
```

Comment:
"fixed bug with short job and long query " ? The onl thing I see is added println...